### PR TITLE
[admin] Admins now have the right to tell people to man up.

### PR DESCRIPTION
**Pull request by ghost**
# General Documentation

### Intent of your Pull Request
This has been tested. I can't really test the global man up without friends. [This is a shameless citadel port.](https://github.com/Citadel-Station-13/Citadel-Station-13/pull/6296)

### Why is this change good for the game?
Man up, and deal with it. Move on.

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Two commands. "Man Up" which shows up in the right click panel. And "Global Man Up" that tells everyone to man up. Using either will make **REALLY BIG TEXT** show up on their screen telling them to man up and deal with it, accompanied by a TTS voice saying the same.

### What should players be aware of when it comes to the changes your PR is implementing?
Admins now have a button to tell you to man up and deal with it.

### What general grouping does this PR fall under? 
Admin additions.

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here.  

# Changelog

Edit this changelog for changes that are noticeable by the players. Remove it if this isn't the case. If you add a name after the ':cl', that name will be used in the changelog. Leave it empty to use your GitHub name. Prefix the PR title with [admin] if it's something admin related. Prefix the PR title with [s] if you are fixing an exploit so it is not announced on discord and the server.

:cl:  
rscadd: admins can now tell people to man up 
soundadd: adds the sound for the man up command 
/:cl:

### CODE REVIEW

**Comment by JamieD1** [comment]
Funny how citadel stole this from my server and now its come back around..

**Comment by JamieD1** [comment]
Fix the admin menu part.

**Comment by ghost** [comment]
@JamieD1 Should be fixed, webedited the category change because it was easy enough.

File: code/modules/admin/admin.dm [hunk 1]
```diff
@@ -988,3 +988,36 @@
...
 		if(string)
 			message_admins("[string]")
+
+/client/proc/cmd_admin_man_up(mob/M in GLOB.mob_list)
+	set category = "Misc"
+	set name = "Man Up"
+
+	if(!M)
+		return
+	if(!check_rights(R_ADMIN))
```
Comment (adamsong) [inline code/modules/admin/admin.dm]: This should probably be R_FUN

**Review by adamsong — CHANGES_REQUESTED** [review]

File: code/modules/admin/admin.dm (hunk 1 above)
Comment (ghost) [reply code/modules/admin/admin.dm]: Will do, give me a moment.

**Review by adamsong — APPROVED** [review]
Code looks good

**Review by nesisknesis — APPROVED** [review]

**Comment by JamieD1** [comment]
Man up global is still special verbs

File: code/modules/admin/admin.dm [hunk 2]
```diff
@@ -988,3 +988,36 @@
...
+	M.playsound_local(M, 'sound/misc/manup.ogg', 50, FALSE, pressure_affected = FALSE)
+
+	log_admin("Man up: [key_name(usr)] told [key_name(M)] to man up")
+	var/message = "<span class='adminnotice'>[key_name_admin(usr)] told [key_name_admin(M)] to man up.</span>"
+	message_admins(message)
+	admin_ticket_log(M, message)
+	SSblackbox.record_feedback("tally", "admin_verb", 1, "Man Up")
+
+/client/proc/cmd_admin_man_up_global()
+	set category = "Special Verbs"
```
Comment (JamieD1) [inline code/modules/admin/admin.dm]:
```suggestion
	set category = "Misc"
```

**Pull request merged**